src/designer/elsa-workflows-studio/src/components/screens/workflow-definition-editor/elsa-workflow-definition-editor-screen/elsa-workflow-definition-editor-screen.tsx

```diff
@@ -673,46 +710,74 @@ export class ElsaWorkflowDefinitionEditorScreen {
-                              : this.renderActivityStatsButton}
-                            onActivityContextMenuButtonClicked={e => this.onActivityContextMenuButtonClicked(e)}
-                            onActivityContextMenuButtonTestClicked={e => this.onActivityContextMenuButtonTestClicked(e)}
-                            activityContextMenu={this.workflowDesignerMode == WorkflowDesignerMode.Edit
-                              ? this.activityContextMenuState
-                              : this.activityContextMenuTestState}
-                            enableMultipleConnectionsFromSingleSource={false}
-                            selectedActivityIds={[this.selectedActivityId]}
-                            onActivitySelected={e => this.onActivitySelected(e)}
-                            onActivityDeselected={e => this.onActivityDeselected(e)}
-                            class="elsa-flex-1"
-                            ref={el => this.designer = el}/>
+        {/* <elsa-designer-tree
+          model={this.workflowModel}
+          mode={this.workflowDesignerMode}
+          layoutDirection={this.layoutDirection}
+          activityContextMenuButton={this.workflowDesignerMode == WorkflowDesignerMode.Edit ? activityContextMenuButton : this.renderActivityStatsButton}
+          onActivityContextMenuButtonClicked={e => this.onActivityContextMenuButtonClicked(e)}
+          onActivityContextMenuButtonTestClicked={e => this.onActivityContextMenuButtonTestClicked(e)}
+          activityContextMenu={this.workflowDesignerMode == WorkflowDesignerMode.Edit ? this.activityContextMenuState : this.activityContextMenuTestState}
+          enableMultipleConnectionsFromSingleSource={false}
+          selectedActivityIds={[this.selectedActivityId]}
+          onActivitySelected={e => this.onActivitySelected(e)}
+          onActivityDeselected={e => this.onActivityDeselected(e)}
+          class="elsa-flex-1"
+          ref={el => (this.designer = el)}
+        /> */}
+        <elsa-designer
+          model={this.workflowModel}
+          mode={this.workflowDesignerMode}
+          layoutDirection={this.layoutDirection}
+          activityContextMenuButton={this.workflowDesignerMode == WorkflowDesignerMode.Edit ? activityContextMenuButton : this.renderActivityStatsButton}
+          onActivityContextMenuButtonClicked={e => this.onActivityContextMenuButtonClicked(e)}
+          onActivityContextMenuButtonTestClicked={e => this.onActivityContextMenuButtonTestClicked(e)}
+          activityContextMenu={this.workflowDesignerMode == WorkflowDesignerMode.Edit ? this.activityContextMenuState : this.activityContextMenuTestState}
+          enableMultipleConnectionsFromSingleSource={false}
+          onActivitySelected={e => this.onActivitySelected(e)}
+          onActivityDeselected={e => this.onActivityDeselected(e)}
+          copiedActivities={this.copiedActivities}
+          class="canvas-container"
+          ref={el => (this.designer = el)}
+        />
```

Comment:
The designer component being used by the system needs to be configurable. Keep in mind that not every user may want to switch to the new designer.